src/ui/buttons/props.js

```diff
@@ -776,15 +776,18 @@ export function normalizeButtonMessage(
     }
   }
 
-  if (color && !values(MESSAGE_COLOR).includes(color)) {
+  if (typeof color !== "undefined" && !values(MESSAGE_COLOR).includes(color)) {
```

Comment:
what prompted the increased strictness on these if statements?

Reply:
we needed this normalization code block to run in the case of the merchant passing in an empty string